x-pack/docs/en/rest-api/security/create-api-keys.asciidoc

```diff
@@ -67,6 +65,51 @@ role descriptor with no privileges. The derived API key can be used for
 authentication; it will not have authority to call {es} APIs.
 
 --
++
+`applications`::: (list) A list of application privilege entries.
+`application` (required):::: (string) The name of the application to which this entry applies
+`privileges`:::: (list) A list of strings, where each element is the name of an application
+privilege or action.
+`resources`:::: (list) A list resources to which the privileges are applied.
+
+`cluster`::: (list) A list of cluster privileges. These privileges define the
+cluster level actions that API keys are able to execute.
+
+`global`::: (object) An object defining global privileges. A global privilege is
+a form of cluster privilege that is request-aware. Support for global privileges
+is currently limited to the management of application privileges.
+This field is optional.
+
+`indices`::: (list) A list of indices permissions entries.
+`field_security`:::: (object) The document fields that the API keys have
+read access to. For more information, see
+<<field-and-document-access-control>>.
+`names` (required):::: (list) A list of indices (or index name patterns) to which the
+permissions in this entry apply.
+`privileges`(required):::: (list) The index level privileges that the API keys
+have on the specified indices.
+`query`:::: A search query that defines the documents the API keys have
+read access to. A document within the specified indices must match this query in
+order for it to be accessible by the API keys.
+
+`metadata`::: (object) Optional meta-data. Within the `metadata` object, keys
+that begin with `_` are reserved for system usage.
+
+`restriction`::: (object) Optional restriction. For more information, see
```

Comment:
> Can we briefly say what restriction is for?
Thanks for the suggestion. Makes sense to describe it. 

> Nit: can we have this after run_as so that this new section is after existing content?

I was thinking the same, but then decided to keep the alphabetical ordering of parameters as this seems to follow the overall convention when listing API parameters.

Reply:
> keep the alphabetical ordering of parameters 

Thanks. I was not aware of that at all.